public/i18n/en.json

```diff
@@ -1,3 +1,16 @@
 {
-  "query-builder-run-query": "Run query"
+  "query-builder-heading": "Simple Query Builder",
+  "query-builder-description": "Welcome to the test system of the Simple Query Builder. Please note that this is a work in progress, not all features are included, and it can sometimes be broken. [$1 Feedback is welcome here.]",
```

Comment:
Are we sure we want _this_ part translated? I think it is fine if that specific copy stays in the query builder for now, at least until we have a decision on https://phabricator.wikimedia.org/T265240

How do you see that?

Reply:
You are right, it would put too much burden and translators on something we are going to throw away soon. It's just weird to see English text at middle of an Arabic interface for example but you're right, we'll migrate it later.

Reply:
Yeah, I sure believe that this is looks weird. Let's worry about it again when we actually have a way to switch the language